src/containers/BellScheduleContainer.js

```diff
@@ -142,7 +143,6 @@ class BellScheduleContainer extends React.PureComponent<Props, State> {
       );
 
       const now = this.state.refreshed;
-      // you can debug with now.hour(11).minute(7)
```

Comment:
you can debug with now.hour(11).minute(7) - Aidan said this was better as a github comment